.github/workflows/checks.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dart-lang/setup-dart@v1
         with:
-          sdk: stable
+          sdk: 2.18.7
```

Comment:
Currently only dart 2 is supported

Will have to spend some time considering what using scip generation on multiple different versions will look like

Reply:
https://github.com/Workiva/scip-dart/issues/51